core/src/main/java/org/mapfish/print/map/style/json/JsonStyleParserHelper.java

```diff
@@ -363,6 +363,18 @@ private void addVendorOptions(final String key, final PJsonObject styleJson, fin
         }
     }
 
+    private double getPxSize(final String size) {
+        if (size.endsWith("px")) {
+            return Double.parseDouble(size.substring(0, size.length() - 2);
+        }
+        else if (size.endsWith("pt")) {
+            // http://www.w3schools.com/cssref/css_units.asp
+            return Double.parseDouble(size.substring(0, size.length() - 2) / 72.0 * 96.0)
```

Comment:
Could you add this conversion to https://github.com/mapfish/mapfish-print/blob/master/core/src/main/java/org/mapfish/print/map/DistanceUnit.java?